utils/codegen/asn1-to-json/tests/test_asn1ToJsonMsg.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+from src.asn1ToJsonMsg import asn1TypeToJsonMsg
+from src.utils.asn1CodeGenerationUtils import parseAsn1Files
+
+class TestASN1ToJsonMsg(unittest.TestCase):
+
+    def setUp(self):
+        self.asn1_files = ["path/to/test_file.asn1"]
```

Comment:
please either add a valid address to the test file, or add a TODO comment to update the address